RELEASE_NOTES.md

```diff
@@ -13,6 +13,10 @@ v1.0.0-alpha.x
 - Removed `toml++` as vendored library. Is now an external dependency
   similar to other external dependencies.
 
+- Updated various Python method argument names to match their C++
```

Comment:
[Consider] It might be best to explicitly list the changed method signatures, for searchability down the line.

Reply:
:thinking: There are quite a few, but I'll have a go and see how it looks.

Reply:
Hmm, personally i'd have it as you've done it there, despite the length, but that's becoming a philosophical point. What _are_ release notes.
I don't think we've got standards on this, so I'll yield to your judgement. What I _wouldn't_ do is compromise and only list a subset just to satisfy my point, go with your gut and commit :P

Reply:
Whatever your choice, merge when ready.

Reply:
Ah, so you've omitted the arguments. but all the args change are for the listed methods. I think that's smart yeah, if you're confused why something isn't working between versions, you'd probably search for the method rather than the arg anyhow.